deploy/helm/kuberhealthy/templates/khcheck-pod-status.yaml

```diff
@@ -44,6 +46,37 @@ spec:
     {{- end }}
     serviceAccountName: pod-status-sa
 ---
+{{- if .Values.check.podStatus.cluster.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: jx-pod-status-check-rb
```

Comment:
can we get rid of the prefix `jx-` and just use `pod-status-check-rb` , `pod-status-role`, etc?